playbooks/lib_backup.yml

```diff
@@ -0,0 +1,94 @@
+---
+- name: build the firewall and vcenter backup server
+  hosts: backup
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/lib_backup/main.yml
+    - ../group_vars/lib_backup/vault.yml
+
+  roles:
+    - role: common
+
+  vars:
+    # You can add more always-allowed users here later if needed
+    allow_ssh_base_users:
+      - pulsys
+
+  tasks:
+    - name: Ensure the backup group exists
+      ansible.builtin.group:
+        name: backup
+        state: present
+
+    - name: Manage users and group membership
+      ansible.builtin.user:
+        name: "{{ item.name }}"
+        password: "{{ item.password }}"
+        shell: /bin/bash
+        groups: backup
+        append: true
+        state: present
+      loop: "{{ backup_users }}"
+
+    - name: create directories with respective ownership
+      ansible.builtin.file:
+        path: "/backup/{{ item.name }}"
+        state: directory
+        mode: "0755"
+        group: backup
+        owner: "{{ item.owner }}"
+      loop:
+        - { name: "palofw", owner: "bkppalofw" }
+        - { name: "vcenter", owner: "bkpvcenter" }
```

Comment:
Use the backup_users variable here, so all data about the users is visible together in the group vars.